openmls/src/group/mls_group/membership.rs

```diff
@@ -93,7 +88,10 @@ impl MlsGroup {
 
     /// Removes members from the group.
     ///
-    /// Members are removed by providing the index of their leaf in the tree.
+    /// Members are removed by providing the member's leaf index.
+    /// The identity is an opaque byte string that is not interpreted by OpenMLS.
```

Comment:
I think this paragraph about what an identity is can go as well, now that we don't take it as an input.